src/plugins/index_pattern_field_editor/public/open_editor.tsx

```diff
@@ -28,7 +28,7 @@ export interface OpenFieldEditorOptions {
   ctx: {
     indexPattern: IndexPattern;
   };
-  onSave?: (field: IndexPatternField) => void;
+  onSave?: (field: IndexPatternField[]) => void;
```

Comment:
To keep consistency with ES we will always return an array, even if it is for one field created.